### ISSUE TITLE
Document import formats (or lack thereof)

### ISSUE THREAD

**Issue by micahellison** [jrnl-org/jrnl#1485]
## Documentation Change

Currently, as far as I can tell, the only format that jrnl supports as an import format is jrnl's own format. And while the `--help` page only lists "jrnl" as one of the --format types next to --import, there are a bunch of other --format arguments listed below it for exporting, so it's a bit confusing (see discussion #1476).

### Affected Page(s) / What Could Be Better?

  - [Command line reference](https://jrnl.sh/en/stable/reference-command-line/) wrongly mentions the export format list in the context of imports
  - [Formats](https://jrnl.sh/en/stable/formats/) could use a mention of this issue

**Comment by xeruf** [comment]
And what if I would contribute an import from another format? Thinking of org-journal right now.

**Comment by micahellison** [comment]
> And what if I would contribute an import from another format? Thinking of org-journal right now.

Hi @xeruf, you're welcome to file an issue and submit a PR for any import format. I think the important thing for imports in particular is that we should have a well-documented spec for any external format that we end up supporting. Ideally, that spec would be versioned and centrally-maintained somewhere.

**Comment by xeruf** [comment]
Yeah that sounds good, I was also wondering where to start in the source code as there seem to be a few possible places.

**Comment by micahellison** [comment]
> Yeah that sounds good, I was also wondering where to start in the source code as there seem to be a few possible places.

For the import code itself, I think a good start would be creating a class similar to: `/jrnl/plugins/jrnl_importer.py`

Your new class should be included in `__importers` in `/jrnl/plugins/__init__.py`

And some tests in `/tests/bdd/features/import.feature` would be essential. I think it could be enough to reuse some of the existing import tests, such as single entry import and multiple entry import.
